readme.md

```diff
@@ -34,7 +34,7 @@ Kick off a run by passing `lighthouse` the URL to audit:
 lighthouse https://airhorner.com/
 ```
 
-Lighthouse will prettyprint a report to CLI. You can control the output format by passing flags.
+Lighthouse will write an HTML report to file. You can control the output format by passing flags.
```

Comment:
"write an HTML report to file" sounds incomplete. "to a file", "to an HTML file"....

What about " By default, Lighthouse writes the report to an HTML file. You can control the output format by passing flags."